### PR TITLE
Add Ammonite JVM properties option

### PR DESCRIPTION
**Pull request by tgodzik**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,6 +100,13 @@
           },
           "markdownDescription": "Optional list of properties to pass along to the Metals server. By default, the environment variable `JAVA_OPTS` and `.jvmopts` file are respected. Each property needs to be a separate item.\n\nExample: `-Dhttps.proxyHost=…`, `-Dhttps.proxyPort=…` or `-Dmetals.statistics=all`"
         },
+        "metals.ammoniteJvmProperties": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "markdownDescription": "Optional list of JVM properties to pass along to the Ammonite server. By default, the environment variable `JAVA_OPTS` and `.jvmopts` file are respected. Each property needs to be a separate item.\n\nExample: `-Xmx4G` or `-Xms1G`"
```
Comment (kpbochenek) [inline package.json:108]: not JAVA_OPTS, JVM_OPTS is currently used in metals

Comment (kpbochenek) [inline package.json:108]: also a 'verify' comment, if you decide to ignore JAVA_OPTS/.jvmopts it needs to be updated here to, if not then all good :)

Comment (tgodzik) [reply package.json]: Removed!

Comment (tgodzik) [reply package.json]: Also updated

**Pull request merged**